packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -0,0 +1,193 @@
+import type { ReactElement } from 'react';
+
+import {
+  currencyAbbreviationsMapping,
+  currencylocaleMapping,
+  currencyPrefixMapping,
+} from './amountTokens';
+import AmountValue, { getAffixFontSize } from './AmountValue';
+import { StyledAmount } from './StyledAmount';
+import { BaseText } from '~components/Typography/BaseText';
+import type { Feedback } from '~tokens/theme/theme';
+import type { BaseTextProps } from '~components/Typography/BaseText/types';
+import BaseBox from '~components/Box/BaseBox';
+import type { TestID } from '~src/_helpers/types';
+import { metaAttribute, MetaConstants } from '~utils';
+
+export const suffixTypes = {
+  HUMANIZE: 'humanize',
+  DECIMALS: 'decimals',
+  NONE: 'none',
+};
+
+type currencyType = 'INR' | 'MYR';
```

Comment:
Lets make it official! We'll use this as the type for when we add a new `currency: Currency` prop
```suggestion
type Currency = 'INR' | 'MYR';
```